scipy/optimize/_linprog_ip.py

```diff
@@ -837,16 +840,16 @@ def eta(g=gamma):
 
 def _linprog_ip(
         c,
-        c0=0,
-        A=None,
-        b=None,
-        callback=None,
-        _T_o=[],
-        alpha0=.99995,
-        beta=0.1,
+        c0,
+        A,
+        b,
+        callback,
+        postsolve_args,
```

Comment:
callback is a parameter of linprog, so it seems higher priority than the options. postsolve_args is needed if there is a callback.